tests/test_markup.py

```diff
@@ -52,6 +52,39 @@ def test_images():
         assert location == expected
 
 
+def test_reference_links_in_xhtml():
+    test_cases = (
+        ('[Named 2019-08-01]', '<a href="notebook:2019-08-01">Named</a>'),
+        ('2019-08-01', '<a href="notebook:2019-08-01">2019-08-01</a>'),
+    )
+
+    for markup, expected_xhtml in test_cases:
+        document = convert(markup, 'xhtml', '/tmp')
+        assert expected_xhtml in document
+
+
+def test_reference_links_in_html():
+    test_cases = (
+        ('[Named 2019-08-01]', '<A HREF="notebook:2019-08-01">Named</A>'),
+        ('2019-08-01', '<A HREF="notebook:2019-08-01">2019-08-01</A>'),
+    )
+
+    for markup, expected_html in test_cases:
+        document = convert(markup, 'html', '/tmp')
+        assert expected_html in document
```

Comment:
I don't think we need to test both html and xhtml. Testing the one that RedNotebook uses should suffice.